tools/ci_build/exclude_unused_ops.py

```diff
@@ -24,6 +24,17 @@
               'com.intel.ai': 'kNGraphDomain',
               'com.xilinx': 'kVitisAIDomain'}
 
+# deprecated ops where the last registration should have an end version.
+# value for each entry is the opset when it was deprecated. end version of last registration should equal value - 1.
+deprecated_ops = {'kOnnxDomain:Scatter': 11,
```

Comment:
>deprecated_ops [](start = 0, length = 14)

Does this mean every time in the future if we deprecate something, we have one more place of code to worry about?

Reply:
Given deprecation is rare I don't think that's a concern vs. the impact of having incorrect kernel registrations.

---
In reply to: [523939316](https://github.com/microsoft/onnxruntime/pull/5817#discussion_r523939316) [](ancestors = 523939316)